source/extensions/matching/actions/format_string/config.cc

```diff
@@ -12,8 +12,9 @@ namespace Matching {
 namespace Actions {
 namespace FormatString {
 
-const Network::FilterChain* ActionImpl::get(const Server::FilterChainsByName& filter_chains_by_name,
-                                            const StreamInfo::StreamInfo& info) const {
+const Network::FilterChain*
+ActionImpl::get(const Server::Configuration::FilterChainsByName& filter_chains_by_name,
+                const StreamInfo::StreamInfo& info) const {
```

Comment:
Out of curiosity, why did this line change?  It *looks* identical but just re-wrapped. Did clang-format dislike the old formatting, I guess?

Reply:
Server::FilterChainsByName ->  Server::Configuration::FilterChainsByName
because it was declared in the envoy/ file which has a different namespace